### PR TITLE
General minor code fixup / overhaul

### PR DESCRIPTION
**Pull request by ToothyDev**
I went through all the main files and tried unifying used concepts, typehints, simplifying imports, etc.
All changes are cosmetic and no behavior was intentionally changed.

If any change I made isn't wanted or could be done even better, do comment

This PR depends on both #64

### CODE REVIEW

**Comment by ToothyDev** [comment]
Well, apparently since I created this branch from another branch and not main, it shows that branch's commits here too - not sure if good or bad but that's how it is now.. welp

**Comment by JoshuaSlui** [comment]
> This PR depends on both #64

on both ?

**Comment by ToothyDev** [comment]
> > This PR depends on both #64
> 
> on both ?

No, just #64, I edited it. Now it really just depends on PyLint being 3.10+

File: utilities/database.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ async def selector(query: str, variables: list):
     return result
 
 
-async def modifier(query: str, variables: list) -> None:
+async def modifier(query: str, variables: list):
```
Comment (JoshuaSlui) [inline utilities/database.py]:
```suggestion
async def modifier(query: str, variables: list) -> None:
```
It does in fact return nothing. You can keep this.

**Review by JoshuaSlui — CHANGES_REQUESTED** [review]

**Comment by JoshuaSlui** [comment]
In the future, i'd like to prevent these PRs, they're great but they shouldn't be necessary.
If you notice small things like this, take care of them while you're working on something.

File: utilities/database.py (hunk 1 above)
Comment (ToothyDev) [reply utilities/database.py]: I looked at the function above, which also has no return value. Should I give that one too then?

Comment (JoshuaSlui) [reply utilities/database.py]: Yes.

Comment (ToothyDev) [reply utilities/database.py]: Well, the return type turns out to be quite messed up and I can't really make that happen on second thought. It'd be extremely generic anyway, so I don't think it's necessary anyway; it depends on the request made and you should roughly expect what kind of data you're getting anyway

Comment (JoshuaSlui) [reply utilities/database.py]: You can just dedicate `selector` as returning a tuple.
It's fine to leave typehinting out with mysql_login, as we cannot verify the return type of that well.

**Review by JoshuaSlui — APPROVED** [review]

**Pull request merged**